log/logs.py

```diff
@@ -0,0 +1,50 @@
+import logging
+import random
+
```

Comment:
It looks like logs can log a game as defined, but where are the uses in other parts of the code? The code here will generate a log to the game you created in play_mock_game, but if the systems runs regularly, no log file will be generated. Also, no example log file.